### PR TITLE
test for loading component

### PR DESCRIPTION
**Pull request by zelalemyetsedaw**
in this pr i implement testing for loading component

### CODE REVIEW

File: __tests__/commons/Loading.test.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from "react";
```
Comment (Segnicho) [inline __tests__/commons/Loading.test.tsx:1]: fix typo docter-detail to doctors-detail

File: components/commons/Loading.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 const LoadingPage: React.FC = () => {
   return (
-    <div className="flex items-center justify-center h-screen">
-      <div className="animate-spin rounded-full h-8 w-8 border-t-2 border-b-2 border-primary"></div>
+    <div role="div" className="flex items-center justify-center h-screen">
```
Comment (Segnicho) [inline components/commons/Loading.tsx:3]: can you clarify why adding role of div as "div" is needed?

File: __tests__/commons/Loading.test.tsx (hunk 1 above)
Comment (Segnicho) [reply __tests__/commons/Loading.test.tsx:1]: or you can make it look like the pages folder by writing your test in doctors/[id].tsx

**Review by Segnicho — CHANGES_REQUESTED** [review]
Looks Good, left you some comments.

File: components/commons/Loading.tsx (hunk 2 above)
Comment (zelalemyetsedaw) [reply components/commons/Loading.tsx:3]: i want to test whether the 2 divs are working properly 
i use getallbyrole to grab them for the testing

File: __tests__/commons/Loading.test.tsx (hunk 1 above)
Comment (zelalemyetsedaw) [reply __tests__/commons/Loading.test.tsx:1]: ok should i put it in the pages/doctors folder

Comment (Segnicho) [reply __tests__/commons/Loading.test.tsx:1]: not pages, but __tests__/doctors/[id].tsx

**Review by Segnicho — APPROVED** [review]
LGTM

**Pull request merged**